### PR TITLE
Configuration for 3-band burnscar algorithm.

### PR DESCRIPTION
**Pull request by james-blitzm**
Add 3-band burnscar configuration (BSI, dNBR, NDVI).

### CODE REVIEW

File: dev/services/alchemist/s2_nrt_bs/s2_nrt_bs.alchemist.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  measurements: ['nbart_nir_1', 'nbart_red_edge_1', 'nbar_blue', 'nbar_red']
+  measurement_renames:
+    nbart_nir_1: nir
+    nbart_red_edge_1: swir2
+    nbar_blue: blue
+    nbar_red: red
+
+  aws_unsigned: True
+  transform: datacube_alchemist.transforms.DeltaNBR_3band
+  transform_args:
```
Comment (alexgleith) [inline dev/services/alchemist/s2_nrt_bs/s2_nrt_bs.alchemist.yaml:14]: Let's drop this if it's empty

File: dev/services/alchemist/s2_nrt_bs/s2_nrt_bs.alchemist.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  inherit_geometry: True
+
+  metadata:
+    product_family: dnbr
+    producer: ga.gov.au
+    dataset_version: 0.0.2
+    collection_number: 3
+    naming_conventions: dea_c3
+  properties:
+    dea:dataset_maturity: interim
```
Comment (alexgleith) [inline dev/services/alchemist/s2_nrt_bs/s2_nrt_bs.alchemist.yaml:40]: Please compare with the S2 WO.

**Review by alexgleith — CHANGES_REQUESTED** [review]

File: dev/services/alchemist/s2_nrt_bs/s2_nrt_bs.alchemist.yaml (hunk 2 above)
Comment (james-blitzm) [reply dev/services/alchemist/s2_nrt_bs/s2_nrt_bs.alchemist.yaml:40]: Updated to match WO version.

File: dev/services/alchemist/s2_nrt_bs/s2_nrt_bs.alchemist.yaml (hunk 1 above)
Comment (james-blitzm) [reply dev/services/alchemist/s2_nrt_bs/s2_nrt_bs.alchemist.yaml]: Updated.

**Review by alexgleith — APPROVED** [review]

**Pull request merged**